pkg/trace/config/config.go

```diff
@@ -79,8 +79,9 @@ type AgentConfig struct {
 	MaxRequestBytes int64 // specifies the maximum allowed request size for incoming trace payloads
 
 	// Writers
-	StatsWriter *WriterConfig
-	TraceWriter *WriterConfig
+	StatsWriter             *WriterConfig
+	TraceWriter             *WriterConfig
+	ConnectionResetInterval time.Duration // frequency at which outgoing connections are reset, in seconds. 0 means no reset is performed
```

Comment:
Extra-nit: the `in-seconds` is not exactly true here. Should probably just be ommited.

Reply:
you're right, I'll fix